### PR TITLE
Allow text color to change on CardPanel and remove default color from CardPanel 

### PR DESCRIPTION
**Pull request by mattsmith14**
The primary change allows the user to override the text color in the CardPanel. Removing teal was because I found it confusing how the CardPanel became teal in the first place.

Thanks!

### CODE REVIEW

**Comment by isaiah** [comment]
Thanks for the PR.

Could you also change the `Card` examples to show how to set text color?

File: src/CardPanel.js [hunk 1]
```diff
@@ -5,12 +5,11 @@ class CardPanel extends React.Component {
   render() {
     let classes = {
       'card-panel': true,
```
Comment (alextrastero) [inline src/CardPanel.js]: unnecessary comma :)

**Comment by alextrastero** [comment]
Also, since it doesn't require any lifecycle, why not make it stateless?

**Comment by mattsmith14** [comment]
@isaiah I have updated the example to show setting text color. Take a look and let me know if that is what you were going for.

@alextrastero I have removed the dangling comma, thanks for pointing that out. I agree on making the component stateless, but I can create a separate pull request for that.

Thanks!

**Comment by mattsmith14** [comment]
I changed my mind and just went ahead and converted CardPanel to a functional component :-).

**Comment by isaiah** [comment]
Thanks.

**Pull request merged**